### PR TITLE
react-native-themis 0.15.2 

### PR DESCRIPTION
**Pull request by radetsky**
Updated versions of dependencies. New minimum versions of iOS and Android are set. Changes are tested on simulators and hardware phones. 

## Checklist

- [x] Change is covered by automated tests
- [x] Changelog is updated (in case of notable or breaking changes)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@
 
 Changes that are currently in development and have not been released yet.
 
+## [0.15.2](https://github.com/cossacklabs/themis/releases/tag/0.15.2), October 03 2023
+
+### Android, ReactNative
```
Comment (vixentael) [inline CHANGELOG.md:9]:
```suggestion
### Android, ReactNative wrappers
```

File: src/wrappers/themis/react-native-themis/BUILD.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ git clone https://github.com/cossacklabs/themis.git
 cd themis 
 carthage bootstrap
 ./scripts/create_xcframeworks.sh
-unzip -d .. src/wrappers/themis/react-native-themis/ios build/xcf_output/themis.xcframework.zip
+unzip -d src/wrappers/themis/react-native-themis/ios build/xcf_output/themis.xcframework.zip
```
Comment (vixentael) [inline src/wrappers/themis/react-native-themis/BUILD.txt:6]: 🤨

File: src/wrappers/themis/react-native-themis/react-native-themis.podspec [hunk 3]
```diff
@@ -9,7 +9,7 @@ Pod::Spec.new do |s|
   s.homepage     = package["homepage"]
   s.license      = package["license"]
   s.authors      = package["author"]
-  s.platforms    = { :ios => "10.0" }
+  s.platforms    = { :ios => "12.4" }
```
Comment (vixentael) [inline src/wrappers/themis/react-native-themis/react-native-themis.podspec:12]: okay!

**Review by vixentael — APPROVED** [review]

**Review by Lagovas — APPROVED** [review]
lgtm

**Pull request merged**